src/helix-ui/styles/components/lists.less

```diff
@@ -26,12 +26,14 @@ hx-dl.hxVertical {
 
   hx-dt {
     #grid.span(12);
+    flex: 0 1 auto;
```

Comment:
To accomplish the vertical style, we have to reset the flex properties in order for the `hx-dt` and `hx-dd` to span the full width of the `hx-def` (the elements shouldn't grow or shrink, and it should extend the full width of the `hx-def`). Given that we're pretty much overriding the `#grid.span()` mixins on lines 8 and 16, we might as well replace the mixins on line 28 and 35 with explicit definitions for the vertical variant.

1. remove the use of the mixins in the vertical variant (we'll replace it with explicit definitions)
2. explicitly negate the `max-width` that gets set by the `#grid.span()` mixins on lines 8 and 16
3. let the contents of the elements dictate their geometry

```less
hx-dl.hxVertical {
  hx-dt,
  hx-dd {
    flex: 0 0 auto; // (3) let content dictate container geometry
    max-width: none; // (2) negate max-width from #grid.span() on lines 8 and 16
  }
  hx-dt {
    // (1) remove #grid.span(12)
  }
  hx-dd {
    // (1) remove #grid.span(12)
  }
}
```